### ISSUE TITLE
Make it easier to run the tests

### ISSUE THREAD

**Issue by robdimsdale** [paketo-buildpacks/stacks#136]
I am currently trying to submit a PR to this repo, but it is not easy to run the tests.

Ideally, I would like a script to run (e.g. `./scripts/test`) but failing that even some documentation on the README would be great.

I'm happy to tackle this once we decide on what we want.

**Comment by robdimsdale** [comment]
I wonder whether this will be naturally resolved when we implement the [Stack Description RFC](https://github.com/paketo-buildpacks/rfcs/blob/main/text/stacks/0003-stack-descriptor.md)

**Comment by sophiewigmore** [comment]
Agreed that this repo will change quite a bit as the stacks descriptor work, https://github.com/paketo-buildpacks/stacks/issues/133, and  https://github.com/paketo-buildpacks/rfcs/issues/197 all land, the structure of the repos will be different. However, we should keep this issue in mind when creating new workflows.

**Comment by ryanmoran** [comment]
I also think the tests themselves should be refactored to align with what we did in the [`jam create-stack` test suite](https://github.com/paketo-buildpacks/jam/blob/main/integration/create_stack_test.go).

That test suite was derived from some refactoring I had done on this other branch of this repo to make the bionic tests a bit easier to deal with. For instance, the [base stack test suite](https://github.com/paketo-buildpacks/stacks/blob/config-file/base/acceptance_test.go) was updated use the new [matchers](https://github.com/paketo-buildpacks/jam/tree/main/integration/matchers) I introduced in the jam integration suite.
